PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,10 @@
+## Description
+
+Describe the changes made and why they were made instead of how they were made.
```

Comment:
I would remove the ` instead of how they were made` section. If someone really wants to provide an overview of the code they changed, especially for a complex PR, I don't think it's necessarily a bad thing.